sway/input/keyboard.c

```diff
@@ -678,9 +685,6 @@ void sway_keyboard_configure(struct sway_keyboard *keyboard) {
 	if (keymap_changed) {
 		ipc_event_input("xkb_keymap",
 			keyboard->seat_device->input_device);
-	} else if (effective_layout_changed) {
-		ipc_event_input("xkb_layout",
```

Comment:
This is not sent anymore. Is this deliberate?

Reply:
I thought so, but it is actually still needed for config reloads. Re-added